pyproject.toml

```diff
@@ -120,6 +120,9 @@ kubernetes = [
 pydantic = [
         "pydantic>=1.10.0,<2.0.0",
 ]
+azure-container-instance = [
+    "apache-airflow-providers-microsoft-azure==8.4.0",
```

Comment:
Could you update this to `apache-airflow-providers-microsoft-azure>=8.4.0` unless there's a reason to pin the provider at 8.4.0

Reply:
Done!